requirements_dev.txt

```diff
@@ -7,4 +7,5 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
-
+decorator==5.1.1
```

Comment:
I added these because when starting a new virtualenv and trying to run the test suite these came up as missing dependencies.